src/lib/pages/frontsite/DocumentRequests/DocumentRequestForm.js

```diff
@@ -203,8 +245,8 @@ class DocumentRequestForm extends Component {
           <>
             <Header as="h1">Request new literature</Header>
             <p>
-              Fill in the form below to request a new literature from the
-              library.
+              Fill in the form below to request to loan, purchase or suggest a
```

Comment:
Yes, I agree about the message, instead of adding it to the config we could add a prop for the `title` and override it with the `parametrize` feature of `react-override`, same  for the `labelSubtitle`, as I did for the `infoMessage` in the [`requestType`](https://github.com/CERNDocumentServer/cds-ils/pull/368/files#diff-562f0ff740e031c21e40f9514b78f33498423e3936cd3655f494226b5001302dR15-R36), WDYT?

Reply:
I think I would go with overriding rather than adding it to the config. It would be more flexible